packages/react/src/main-nav/NavContainer.tsx

```diff
@@ -66,7 +62,6 @@ export function NavContainer({
 				'&:focus': { outline: 'none' },
 				[localPaletteVars.linkHoverBg]: backgroundColorMap[hover],
 				[localPaletteVars.linkActiveBg]: backgroundColorMap[background],
-				[localPaletteVars.bottomBar]: boxPalette.accent,
```

Comment:
Similar to `Footer`, in the `MainNav` component we only ever set the border colour to accent so there is no need for this local palette variable.